### ISSUE TITLE
Did you try Focal loss, how about its performance ?

### ISSUE THREAD

**Issue by YunYang1994** [ShiqiYu/libfacedetection.train#81]
Did you try Focal loss, how about its performance ?

**Comment by Wwupup** [comment]
HelIo @YunYang1994 , have previously attempted to directly use focal loss as the classification loss, using the recommended hyperparameters from the paper, but the improvement seemed insignificant.
